doc/content/language.md

```diff
@@ -1325,3 +1325,33 @@ output.icecast(%mp3, host="localhost", port=8000,
 ```
 
 so that passwords are not shown in the main script.
+
+### Code comments
+
+Comments can be added to your code in two ways:
+
+_Multi-line comments_ are comments that can span multiple lines. They are delimitated
+by the sequence of characters `#<` at the beginning and `>#` at the end. Anything
+in between those two sequences is considered code comment. Nested code comments
+are also accepted:
+
+```liquidsoap
+#<------- BEGIN CODE COMMENT ----#
+    This is a comment
+
+# This is also a comment
+
+  #< This is a nested code comment >#
+
+#--------- END CODE COMMENT ----->#
+```
+
+_Single-line comments_ are comments that are limited to the current line. Such comments
+are started with the character `#` without a following `<`. Anything after the initial
+`#` character and until the end of the line is considered code comment:
```

Comment:
Single-line comments after multiline comments?

Reply:
It seems more logical this way. How would you describe the single line comment syntax without mentioning multi-line comments?

Reply:
Previously in multi-line comments.
> `# This is also a comment`